src/test/java/org/folio/circulation/domain/policy/FixedLoanPolicyRenewalDueDateCalculationTests.java

```diff
@@ -223,6 +233,64 @@ public void shouldUseAlternateScheduleWhenAvailable() {
     assertThat(result.value().getDueDate(), is(expectedSchedule.due));
   }
 
+  @Test
+  public void shouldApplyAlternateScheduleWhenQueuedRequestIsHoldAndFixed() {
+    final Period alternateCheckoutLoanPeriod = Period.from(2, "Weeks");
+    final DateTime systemTime = DateTime.now();
+
+    LoanPolicy loanPolicy = LoanPolicy.from(new LoanPolicyBuilder()
+      .fixed(UUID.randomUUID())
+      .withAlternateCheckoutLoanPeriod(alternateCheckoutLoanPeriod)
+      .create())
+      .withDueDateSchedules(new FixedDueDateSchedulesBuilder()
+        .addSchedule(FixedDueDateSchedule.wholeYear(systemTime.getYear()))
+        .create());
+
+    Item item = Item.from(
+      new ItemBuilder()
+        .checkOut()
+        .withId(UUID.randomUUID())
+        .create());
+    Loan loan = Loan.from(
+      new LoanBuilder()
+        .withItemId(UUID.fromString(item.getItemId()))
+        .withLoanDate(systemTime)
+        .create());
+
+    Request requestOne = Request.from(new RequestBuilder()
+      .withId(UUID.randomUUID())
+      .withItemId(UUID.fromString(loan.getItemId()))
+      .withPosition(1)
+      .create());
+
+    Request requestTwo = Request.from(new RequestBuilder()
+      .withId(UUID.randomUUID())
+      .withStatus(RequestStatus.OPEN_NOT_YET_FILLED.getValue())
+      .hold()
+      .withItemId(UUID.fromString(loan.getItemId()))
+      .withPosition(2)
+      .create());
+
+    RequestQueue requestQueue = new RequestQueue(asList(requestOne, requestTwo));
+    Result<DateTime> initialDueDateResult = loanPolicy.calculateInitialDueDate(loan, requestQueue);
+
+    String key = "alternateCheckoutLoanPeriod";
+    DateTime expectedDueDate = alternateCheckoutLoanPeriod.addTo(
+        systemTime,
+        () -> errorForLoanPeriod(format("the \"%s\" is not recognized", key)),
+        interval -> errorForLoanPeriod(format("the interval \"%s\" in \"%s\" is not recognized", interval, key)),
+        dur -> errorForLoanPeriod(format("the duration \"%s\" in \"%s\" is invalid", dur, key)))
+          .value();
+    Long result = initialDueDateResult.value().getMillis();
+    Long expected = expectedDueDate.getMillis();
+    assertThat(result.doubleValue(), closeTo(expected.doubleValue(), 60000));
```

Comment:
What does this refer to?

Personally, I would find it more readable if the assertions referred to the date and time that was expected in a more human friendly format (I believe the other tests do this)